trulens_eval/trulens_eval/tests/groundedness_smoke_tests.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    f_groundtruth = Feedback(ground_truth.numeric_difference, name = "Groundedness Smoke Test").on(Select.Record.calls[0].args.args[0]).on(Select.Record.calls[0].args.args[1]).on_output()
</pre><p>Can we call this MAE instead? Would also apply to relevance smoke tests.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>sg - will refactor</p>